src/iac_scan_runner/utils.py

```diff
@@ -97,3 +97,18 @@ def write_html_to_file(file_name: str, output_value: str):
             text_file.write(output_value)
     except Exception as e:
         raise Exception(f"Error storing HTML to file: {str(e)}.")           
+
+
+def file_to_string(file_path: str):
```

Comment:
Missing return type.

```suggestion
def file_to_string(file_path: str) -> str:
```

Reply:
I definitely forgot to add it, gonna fix it